src/modules/mana/sagas.ts

```diff
@@ -496,15 +494,16 @@ export const importWithdrawalErrors = {
   other: (msg: string) => formatImportWithdrawalError(msg),
 }
 
-function* handleImportWithdrawalRequest(action: ImportWithdrawalRequestAction) {
+export function* handleImportWithdrawalRequest(action: ImportWithdrawalRequestAction) {
   const {
     payload: { txHash },
   } = action
 
   try {
     const address: string | undefined = yield select(getAddress)
-    const provider: Provider = yield call(() =>
-      getNetworkProvider(getChainIdByNetwork(Network.MATIC))
+    const provider: Provider = yield call(
+      getNetworkProvider,
+      getChainIdByNetwork(Network.MATIC)
     )
 
     const transaction:
```

Comment:
The call to send that is used below can surely result into an exception. A new test case must be added to assert that the sagas will do what we want in this case.

Reply:
This was not addressed.